<?xml version="1.0"?>
<launch>
    <node pkg="dashboard" type="dashboard.py" name="dashboard_node" cwd="node">
        <param name="window_dimensions" value="(800, 800)" />
        <!--
        <param name="background_color" value="(0, 0, 0)" />
        <param name="text_color" value="(0, 0, 0)" />
        <param name="text_shadow_color" value="(0, 0, 0)" />
        <param name="ego_color" value="(0, 0, 0)" />
        <param name="final_waypoints_color" value="(0, 0, 0)" />
        <param name="base_waypoints_color" value="(0, 0, 0)" />
        -->
    </node>
</launch>
